packages/terra-date-time-picker/src/_TimeClarification.jsx

```diff
@@ -13,7 +13,7 @@ const propTypes = {
   /**
    * An ISO 8601 date time with the ambiguous hour.
    */
-  ambiguousDateTime: PropTypes.string,
+  ambiguousDateTime: PropTypes.object,
```

Comment:
Is the reason to change the prop type to be a moment object because we can get the timezone from it? Since this is a private class it's not as big of an issue but since this is a first-class prop I feel we shouldn't expect the prop type to be a third party object. Would it be better to keep the type as a string and add a new `timezone` prop instead? If we do decide to change to object, the prop description should be updated.

Reply:
Updated here 1e47747132f83358d17d2aaed1f98c6bd4a5cfaf